rskj-core/src/main/java/co/rsk/peg/utils/BridgeEventLoggerImpl.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2017 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package co.rsk.peg.utils;
+
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.core.BtcTransaction;
+import co.rsk.config.BridgeConstants;
+import co.rsk.peg.Bridge;
+import co.rsk.peg.Federation;
+import org.ethereum.core.Block;
+import org.ethereum.core.Transaction;
+import org.ethereum.rpc.TypeConverter;
+import org.ethereum.util.RLP;
+import org.ethereum.vm.DataWord;
+import org.ethereum.vm.LogInfo;
+import org.ethereum.vm.PrecompiledContracts;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Responsible for logging events triggered by BridgeContract.
+ *
+ * @author martin.medina
+ */
+public class BridgeEventLoggerImpl implements BridgeEventLogger {
+
+    private static final byte[] BRIDGE_CONTRACT_ADDRESS = TypeConverter.stringToByteArray(PrecompiledContracts.BRIDGE_ADDR);
+
+    private final BridgeConstants bridgeConstants;
+
+    private List<LogInfo> logs;
+
+    public BridgeEventLoggerImpl(BridgeConstants bridgeConstants, List<LogInfo> logs) {
+        this.bridgeConstants = bridgeConstants;
+        this.logs = logs;
+    }
+
+    public List<LogInfo> getLogs() {
+        return logs;
+    }
+
+    public void logUpdateCollections(Transaction rskTx) {
+        this.logs.add(
+                new LogInfo(BRIDGE_CONTRACT_ADDRESS,
+                            Collections.singletonList(Bridge.UPDATE_COLLECTIONS_TOPIC),
+                            RLP.encodeElement(rskTx.getSender())
+                )
+        );
+    }
+
+    public void losAddSignature(BtcECKey federatorPublicKey, BtcTransaction btcTx, byte[] rskTxHash) {
```

Comment:
Looks like there is a typo in this method name

Reply:
My fingers bad, fixed!